src/cljs/rems/spa.cljs

```diff
@@ -332,11 +330,13 @@
 
 (defn page []
   (let [page-id @(rf/subscribe [:page])
-        grab-focus? @(rf/subscribe [::grab-focus?])]
+        grab-focus? @(rf/subscribe [::grab-focus?])
+        theme @(rf/subscribe [:theme])]
     (lazy-load-data!)
     [:div
      [nav/navigation-widget]
-     [logo]
+     (when (or (= page-id :home) (str/blank? (:logo-name-navigation theme)))
+       [nav/logo])
```

Comment:
This test does not work. The hero version of the logo should appear if it is configured on all pages and not just home. It should also not depend on the navbar logo at all. Also this test is wrong (see the previous comment).

Also I'm not sure why the logo component makes sense to be in the navbar namespace where it has been moved in this PR. I think it should probably be in the atoms.